gqlalchemy/query_builder.py

```diff
@@ -336,6 +337,17 @@ def construct_query(self) -> str:
         return f" ORDER BY {self.properties} "
 
 
+class OrderByDescPartialQuery(PartialQuery):
+    def __init__(self, properties: str):
+        super().__init__(DeclarativeBaseTypes.DESC)
+
+        self.properties = properties
+
+    def construct_query(self) -> str:
+        """Creates a ORDER BY statement Cypher partial query."""
+        return OrderByPartialQuery(self.properties).construct_query() + "DESC "
```

Comment:
Because it needs to have one space extra at the end, I will try solving that in another way. I'll let you know once I fix it.

Reply:
I feel that most of the code needs to be rewritten.

I would write something like:

`return f"{OrderByPartialQuery(self.properties).construct_query()}{DeclarativeBaseTypes.DESC} "`

Reply:
As Ivan noticed above, I was wrong with putting the DESC in DeclarativeBaseTypes. Therefore, we don't have constants for any of the strings in the query. That's why I need to use "DESC ", like "ORDER BY "which was used above in `order_by()` method. But I agree, this will need to be changed sometimes.

Reply:
With the current code base, the correct would be:

`return f"{OrderByPartialQuery(self.properties).construct_query()}DESC "`

Reply:
I pushed new commit, let me know if that's fine.

Reply:
I pushed the version with the inheritance, we can discuss it tomorrow. If it's not okay, I can revert it to the original version.